### ISSUE TITLE
401 error when publishing new plugin version

### ISSUE THREAD

**Issue by markrekveld** [jenkins-infra/helpdesk#4410]
### Service(s)

GitHub

### Summary

I use the `cd` workflow action in Github to publish a new release of my plugin.
But now I'm blocked with this 401 error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.1.3:deploy (default-deploy) on project jira-integration: Failed to deploy artifacts: Could not transfer artifact io.jenkins.plugins:jira-integration:pom:5.5.0-48.vedfd7e01fc65 from/to maven.jenkins-ci.org (https://repo.jenkins-ci.org/releases/): status code: 401, reason phrase:  (401) -> [Help 1]
```

Can you please assist in refreshing the `MAVEN_TOKEN` of my plugin `jira-integration` https://github.com/jenkinsci/jira-integration-plugin

**Comment by markrekveld** [comment]
Thanks for your update @dduportal

**Comment by dduportal** [comment]
- We found and fixed the issue. Agents are back, I've triggered a new build of the RPU jobs: expect the token `MAVEN_TOKEN` in ~30 min (I'll check the job which updates it on my side).
- For audit trail: 
  - We found the root cause: I made a mistake in https://github.com/jenkins-infra/jenkins-infra/pull/3737 by changing the shebang of the init script from `#!/bin/sh` to `!#/bin/bash`.
  - This change works with inbound agent (on ci.jenkins.io) but does not seem to work with SSH agents (on trusted.ci), preventing agents to be started.
  - Hot fixed with https://github.com/jenkins-infra/jenkins-infra/commit/3517b47da0f75fb1124d68e321aad9057671645a and deployed with success
